pkg/features/kube_features.go

```diff
@@ -997,7 +997,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	MemoryQoS: {Default: false, PreRelease: featuregate.Alpha},
 
-	MinDomainsInPodTopologySpread: {Default: false, PreRelease: featuregate.Beta},
+	MinDomainsInPodTopologySpread: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
Thanks for the catch, LGTM, but cc @sanposhiho for confirm.

Reply:
Oh, sorry for this. It's my mistake on https://github.com/kubernetes/kubernetes/pull/110388.
Thanks @mengjiao-liu for the good catch.